src/state/chat/reducer.js

```diff
@@ -23,6 +22,9 @@ import {
 } from '../action-types';
 import { HAPPYCHAT_CHAT_STATUS_DEFAULT } from '../constants';
 
+const OBVS_UNIX_SEC_DATE = 1700000000;
+export const upscaleTimePrecision = ( time ) => ( time < OBVS_UNIX_SEC_DATE ? time * 1000 : time );
```

Comment:
What about naming this `maybeUpscaleTimePrecision` so it's more obvious what it does?